source/SIL.AppBuilder.Portal.Frontend/src/lib/routing.tsx

```diff
@@ -17,3 +20,13 @@ export function withResetScroll(WrappedComponent) {
     </ScrollToTop>
   );
 }
+
+export function Route(passedProps: any /* RouteProps */) {
```

Comment:
Almost all project ruotes have been swapped with this route wrapper as it provides some decent error handling without totally breaking the app when an error occurs